README.md

```diff
@@ -1,8 +1,7 @@
-![CI](https://github.com/crossplane/crossplane/workflows/CI/badge.svg) [![GitHub release](https://img.shields.io/github/release/crossplane/crossplane/all.svg?style=flat-square)](https://github.com/crossplane/crossplane/releases) [![Docker Pulls](https://img.shields.io/docker/pulls/crossplane/crossplane.svg)](https://hub.docker.com/r/crossplane/crossplane) [![Go Report Card](https://goreportcard.com/badge/github.com/crossplane/crossplane)](https://goreportcard.com/report/github.com/crossplane/crossplane) [![Slack](https://slack.crossplane.io/badge.svg)](https://slack.crossplane.io) [![Twitter Follow](https://img.shields.io/twitter/follow/crossplane_io.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=crossplane_io&user_id=788180534543339520)
```

Comment:
The previous badge we were using included a count of Slack members, I believe. @jbw976 do you know how this badge worked?

Reply:
No, it wasn't intentional. I only updated the broken link for the Slack badge and didn't modify the code.

Reply:
Hello **@negz** sir, I've noticed that the **PR template is currently unavailable to me**. It would greatly assist me if you could provide the PR template, as I intend to make edits to it and proceed with the commit. Your assistance in this matter would be highly appreciated.

Reply:
> The previous badge we were using included a count of Slack members, I believe. @jbw976 do you know how this badge worked?

That was functionality provided by https://github.com/rauchg/slackin - we're no longer able to get/use that now because of https://github.com/crossplane/crossplane/issues/3907.  We may be able to have similar functionality when the long term replacement for slackin is in place, but for now this broken image fix is sufficient!

Reply:
@Rohit312001 you shouldn't need access to change/update the PR template itself, which is stored at https://github.com/crossplane/crossplane/blob/master/.github/PULL_REQUEST_TEMPLATE.md.  That template is automatically shown to you when you first open the PR, but perhaps you deleted that content at that time.

Having details and surrounding context in the PR bodies is helpful to understand all the changes going into Crossplane, so to aid with this, I will edit the PR body here to put the template back in place, and then you should be able to edit the PR body to fill out the template completely 🙇‍♂️